docs/howto/UpdateImports.md

```diff
@@ -0,0 +1,176 @@
+# Update Imports Workflow
```

Comment:
Remove this file from PR (duplicate with update-import.md)

Reply:
Whoops, my bad, didn't realise that got copied in too >.<